src/client/Microsoft.Identity.Client/Platforms/Android/Broker/BrokerConstants.cs

```diff
@@ -182,5 +182,19 @@ internal static class BrokerConstants
         public const string BrokerResultV2 = "broker_result_v2";
         public const string BrokerRequestV2Success = "broker_request_v2_success";
         public const string BrokerUnknownErrorCode = "unknown_error";
+
+        public const string CompanyPortalPackageName = "com.microsoft.windowsintune.companyportal";
+        public const string CompanyPortalContentProviderUri = "content://com.microsoft.windowsintune.companyportal.microsoft.identity.broker";
+        public const string MsAuthenticatorContentProviderUri = "content://com.azure.authenticator.microsoft.identity.broker";
+    }
+
+    //Content provider operation
+    internal enum ContentResolverOperation
+    {
+        hello,
```

Comment:
nit: add a small comment on what "hello" is, as it's specific to the android broker.

Reply:
I added it above the enum because the ContentResolverOperation is only used by android